Makefile

```diff
@@ -18,14 +18,14 @@ tidy:
 
 .PHONY:
 up:
-	docker compose up --build -d;
+	docker-compose up --build -d;
```

Comment:
We can ensure people are using v2 of docker compose if we leave out the dash: https://www.docker.com/blog/announcing-compose-v2-general-availability/

something to think about

Reply:
I have fixed this in the newest push.